service/core/client/src/main/scala/com/lightbend/lagom/internal/client/WebSocketClient.scala

```diff
@@ -399,3 +398,16 @@ object WebSocketClient {
 class WebSocketException(s: String, th: Throwable) extends java.io.IOException(s, th) {
   def this(s: String) = this(s, null)
 }
+
+sealed trait WebSocketClientConfig {
+  def maxFrameLength: Int
+}
+
+object WebSocketClientConfig {
```

Comment:
I think the new trait and object should be `private[lagom]`

Reply:
I think the `trait` is exposed via DI (Guice or think cake).

Reply:
Indeed, it is...and that's actually not working well on that part (whence the failed CI checks)

I have made a ` WebSocketConfigProvider` but I'm having some trouble to find where I should bind it as a provider for `WebSocketConfig`. I was thinking about `ServiceClientGuiceSupport` (in [bindClient](https://github.com/lagom/lagom/blob/4de3e16251852572c703babb1bfd3d9a386dfc3d/service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/ServiceClientGuiceSupport.java#L26) ?) but I am quite unsure and I don't know how to get the base `Config` from there.

If you have some hints about this, that would be great.

Reply:
OK then in that case, I think it should not be in the `internal` package. Either it should be internal and `private[lagom]` or it should be part of the documented API, not in an `internal` package, and public.

Reply:
Looking at it more closely, I think it _should_ be `private[lagom]` and `internal`, but the `webSocketClientConfig` and `scaladslWebSocketClient` definitions in `LagomServiceClientComponents` should _also_ be `private`. They are only used in the construction of `ScaladslServiceClient` and should not be exposed to end users.